root/static/scripts/relationship-editor/common/multiselect.js

```diff
@@ -95,8 +95,12 @@ import deferFocus from '../../edit/utility/deferFocus';
 
                 var visible = matchIndex(option, term) >= 0 && (
                     linkedEntities.link_attribute_type[typeGID].creditable ||
-                    _.findIndex(selected, function (a) { return a.type.gid === typeGID }) < 0
-                );
+                    _.findIndex(
+                        selected,
+                        function (a) {
+                            return a.type.gid === typeGID;
+                        },
+                    ) < 0);
```

Comment:
I think keeping it on one line and using an arrow function would be much more readable here
```JS
_.findIndex(selected, a => a.type.gid === typeGID) < 0
```